app/src/sharedTest/java/org/oppia/app/utility/ClickActions.kt

```diff
@@ -0,0 +1,28 @@
+package org.oppia.app.utility
+
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.CoordinatesProvider
+import androidx.test.espresso.action.GeneralClickAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Tap
+
+fun clickAtXY(pctX: Float, pctY: Float): ViewAction {
```

Comment:
Please add documentation to _all_ public APIs unless they are trivial (basically only standard setters & getters--everything else needs to have documentation to provide context to other team members).

Reply:
done